.github/workflows/main.yaml

```diff
@@ -170,6 +170,13 @@ jobs:
 
       - run: make e2e
 
+      - uses: actions/upload-artifact@v3
+        with:
+          name: my-artifact
+          path: |
+            /tmp/cloudcore.log
+            /tmp/edgecore.log
```

Comment:
> In the summary of the action, at the bottom of the web page.
For example: https://github.com/kubeedge/kubeedge/actions/runs/2418779085
This maybe benefit the debug when e2e tests fail and developers want to look up edgecore or cloudcore logs
Is it necessary to keep this? @fisherxu

Looks good, and seems the name can be `e2e-test-logs`? Other e2e tests like `keadm e2e`, we can also upload the logs?

Reply:
keadm e2e logs will be logged by systemd, so we can only upload `e2e` logs